ldm/modules/encoders/modules.py

```diff
@@ -136,7 +136,7 @@ def encode(self, x):
 
 class FrozenCLIPEmbedder(AbstractEncoder):
     """Uses the CLIP transformer encoder for text (from Hugging Face)"""
-    def __init__(self, version="openai/clip-vit-large-patch14", device="cuda", max_length=77):
+    def __init__(self, version="openai/clip-vit-large-patch14", device="cuda:1", max_length=77):
```

Comment:
As far as I know using just `device="cuda"` works even with a single GPU as it will get the first GPU available but when using `device="cuda:1"` it will only work if you have more than one GPU as it will take the second GPU but not if you only have 1.